### DISCUSSION TITLE
Embedding juggle on a website

### DISCUSSION THREAD

**Discussion by ladanuzhna** [HEmile/juggl#60] [Q&A] [answered]
Hi! Thank you for building this :)
I was just wondering whether there is a way to export and embed these graphs? I want to embed a graph on a website for my research but Obsidian & Roam do not seem to offer this capability.

**Comment by HEmile** [comment, marked as answer]
Hi, thanks for your question! I have not tried this, but what you might be able to do (would require some coding): 
- Save your graph in the workspace mode
- Copy `.obsidian/plugins/juggl/<name of your saved graph>/graph.json` to your server
- Add cytoscape.js to your website
- In cytoscape.js, load graph.json with `cy.json()` (see https://js.cytoscape.org/#cy.json)

However, extra features added by Juggl such as the hover preview will not work, you'd just be able to see the graph. Embedding Juggl itself is not (currently) possible, only a snapshot of it.
